### ISSUE TITLE
[AXE-CORE]: Disability increase review application definition lists must be structured correctly

### ISSUE THREAD

**Issue by aeltarhoni** [department-of-veterans-affairs/va.gov-team#1709]
@1Copenut commented on [Mon Nov 12 2018](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15087)

## Issue
The definition lists on the review page of our apply for increase form have a number of `<div>` elements inside the opening `<dl>` tag, and it's causing a number of errors. Screenshot attached below.

## Issue Type
* Defect

## Acceptance Criteria
* As a tester, I don't want to see any definition list errors when I run subsequent axe plugin scans.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/disability-benefits/apply/form-526-disability-claim/veteran-details/address-information
* login: ID.me
* user: vets.gov.user+228@gmail.com (Ask Trevor Pierce for PW if necessary)

## Steps to Recreate
1. Log into staging.va.gov with ID.me (test user listed above in Environment)
2. Enter `https://staging.va.gov/disability-benefits/apply/form-526-disability-claim/review-and-submit` in browser. (Might have to step through the form again, entering data)
3. Open Developer Tools and click the axe `ANALYZE` button (Assume latest [aXe 3.6.x](https://www.deque.com/axe) browser plugin is installed)
4. Make sure the dropdown on the left of the aXe developer tools panel says **All issues** or **Violations**
5. Verify the error exists

## Pages Affected
* https://staging.va.gov/disability-benefits/apply/form-526-disability-claim/review-and-submit

## Possible Fix
Definition lists must be properly formed. Very truncated example below.

```html
<dl class="review">
  <dt>Phone number</dt>
  <dd>
    <span>(555) 666-4444</span>
  </dd>
</dl>
```

## WCAG or Vendor Guidance 
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

---

![screen shot 2018-11-12 at 4 09 35 pm](https://user-images.githubusercontent.com/934879/48378105-5f688100-e695-11e8-9cc3-f5b6f728bd64.png)

---

@1Copenut commented on [Thu Feb 07 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15087#issuecomment-461442848)

This will probably have a US-forms-system dependency. Can write a separate ticket on that repo and reference it as needed.

**Comment by Mottie** [comment]
This issue has been resolved

**Comment by 1Copenut** [comment]
@Mottie Is there a PR I can take a look at for this issue?

**Comment by Mottie** [comment]
I didn't find a PR, but the possible fix HTML shared above is in place when I looked. I'll try to find a PR/commit.

**Comment by Mottie** [comment]
@1Copenut I'm still learning the code base... I spent a bit too much time trying to find the PR. Sorry, no luck. Do you want me to keep looking?

**Comment by 1Copenut** [comment]
@Mottie No problem. I investigated the issue on a few other forms and opened a sitewide ticket related to other instances of the issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/1943

Thank you for following up.

**Comment by Mottie** [comment]
Oh, I think I misunderstood... I was looking for the `<span>` wrapping the content in the `<dd>`.

**Comment by 1Copenut** [comment]
I'd say that's on me for not explaining the issue as well as I could. I first saw this issue in the 526 form, but after quick review yesterday, it's definitely a wider item that the Tools team should investigate. My hunch is the issue is in the forms system markup, so a fix will have to happen there, be pushed into the code base, and QA'ed for accuracy in multiple places.
